mii/legacy/examples/local/conversational-example.py

```diff
@@ -11,4 +11,4 @@
 
 print(f"Deploying {name}...")
 
-mii.deploy(task='conversational', model=name, deployment_name=name + "_deployment")
+mii.deploy(task='question-answering', model=name, deployment_name=name + "_deployment")
```

Comment:
`TextGenerationPipeline` is recommended as replacement for `ConversationalPipeline`, so you likely want `task='text-generation'` here.

Reply:
Good catch, updated.